tests/h/factories_test.py

```diff
@@ -15,78 +15,30 @@ def test_id(self):
         annotation_1 = factories.Annotation()
         annotation_2 = factories.Annotation()
 
-        assert annotation_1.get("id")
-        assert annotation_2.get("id")
-        assert annotation_1["id"] != annotation_2["id"]
+        assert annotation_1.id
+        assert annotation_2.id
+        assert annotation_1.id != annotation_2.id
 
     def test_text(self):
         """Each annotation should have unique note text."""
         annotation_1 = factories.Annotation()
         annotation_2 = factories.Annotation()
 
-        assert annotation_1.get("text")
-        assert annotation_2.get("text")
-        assert annotation_1["text"] != annotation_2["text"]
+        assert annotation_1.text
+        assert annotation_2.text
+        assert annotation_1.text != annotation_2.text
 
-    def test_custom_user(self):
-        """A custom username should be used in the user field."""
-        annotation = factories.Annotation(username="bobo")
-        assert "bobo" in annotation["user"]
-        assert "username" not in annotation
-
-    def test_created_date(self):
-        """Annotations should have a created date from the current time."""
-        before = datetime.datetime.now()
-
-        annotation = factories.Annotation()
-
-        after = datetime.datetime.now()
-        created = datetime.datetime.strptime(
-            annotation["created"], "%Y-%m-%dT%H:%M:%S.%f")
-
-        assert before < created < after
-
-    def test_updated_date(self):
-        """Annotations should have an updated date from the current time."""
-        before = datetime.datetime.now()
-
-        annotation = factories.Annotation()
-
-        after = datetime.datetime.now()
-        updated = datetime.datetime.strptime(
-            annotation["updated"], "%Y-%m-%dT%H:%M:%S.%f")
-
-        assert before < updated < after
```

Comment:
There's no longer anything about created and updated times in any of the factory classes. These columns have defaults in the db, so it's a feature of the ORM classes they you don't have to pass them in and values will juist be generated, the factory classes just use that.